src/main/java/me/totalfreedom/totalfreedommod/util/FUtil.java

```diff
@@ -53,12 +57,32 @@
             ChatColor.LIGHT_PURPLE,
             ChatColor.YELLOW);
 
+    public static final List<ChatColor> CHAT_RAINBOW = Arrays.asList(
```

Comment:
So im guessing you never bothered to watch the video I uploaded? The video is now on YouTube https://www.youtube.com/watch?v=qpMoEUUMx58&feature=youtu.be